server/src/test/java/org/elasticsearch/indices/cluster/ClusterStateChanges.java

```diff
@@ -462,7 +462,7 @@ private ClusterState executeClusterStateUpdateTask(ClusterState state, Runnable
             ClusterStateUpdateTask task = (ClusterStateUpdateTask) invocationOnMock.getArguments()[1];
             result[0] = task.execute(state);
             return null;
-        }).when(clusterService).submitStateUpdateTask(anyString(), any(ClusterStateUpdateTask.class));
+        }).when(clusterService).submitStateUpdateTask(anyString(), any(ClusterStateUpdateTask.class), ClusterStateTaskExecutor.unbatched());
```

Comment:
And here

Reply:
Looks like there might be number of other tests affected by this, lets wait for the build to have a full list of them to fix.